apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/constants.ts

```diff
@@ -1 +1,11 @@
 export const STORE_KEY = 'automattic/block-editor-nav-sidebar';
+
+declare global {
+	interface Window {
+		wpcomBlockEditorNavSidebar: {
+			homeUrl: string;
+		};
+	}
+}
+
+export const SITE_HOME_URL = window.wpcomBlockEditorNavSidebar.homeUrl;
```

Comment:
hmmm when I first applied the diff, I was getting an error, `cannot read homeUrl of undefined` This would be due to caching somehow because after waiting 5 minutes and refreshing the page again, I no longer saw the error. I wonder if it's worth adding a null check here?

Reply:
Wouldn't hurt. If `SITE_HOME_URL` ended up undefined then it just wouldn't render in the sidebar, which seems like a fairly graceful failure.